documentation/src/components/layout.css

```diff
@@ -73,8 +80,68 @@ header svg {
     height: 100%;
 }
 
+.manage-theme {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.manage-theme label {
+    margin-left: var(--spectrum-global-dimension-size-400);
+}
+
 @media screen and (min-width: 961px) {
     header {
         display: none;
     }
 }
+
+label {
```

Comment:
Manually apply `label` styling from Spectrum CSS while we have yet to ship an implementation of that pattern. Tracked in https://github.com/adobe/spectrum-web-components/issues/475